### ISSUE TITLE
Connection Closed

### ISSUE THREAD

**Issue by prabinpps** [owncloud/core#26121]
### Steps to reproduce
1. Synchronize files with the owncloud client
2. Successful Login

### Expected behaviour
Synchronize files with the owncloud client 

### Actual behaviour
Connection Closed
Shows "Page isn't working" after login

### Server configuration
**Operating system**: CentOS 7

**Web server:** Apache

**Database:** Mariadb

**PHP version:** PHP 5.6.24

**ownCloud version:** ownCloud 9.1.0 (stable)

**Updated from an older ownCloud or fresh install:** Fresh

**Where did you install ownCloud from:** Owncloud repos

**Signing status (ownCloud 9.0 and above):**

```
Recent oc log:
found.","level":3,"time":"2016-09-16T11:30:18+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"homraj@sagar.com.np"}
{"reqId":"V9vXyjmEtLKd8BuSChxYhQAAAAU","remoteAddr":"192.168.0.169","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:19+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"basudev@sagar.com.np"}
{"reqId":"V9vXy4wHRosyWq0g3fFYRgAAAA0","remoteAddr":"192.168.0.222","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:19+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"bijaykc@sagar.com.np"}
{"reqId":"V9vXy4wHRosyWq0g3fFYRgAAAA0","remoteAddr":"192.168.0.222","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:19+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"bijaykc@sagar.com.np"}
{"reqId":"V9vXy4wHRosyWq0g3fFYRgAAAA0","remoteAddr":"192.168.0.222","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:20+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"bijaykc@sagar.com.np"}
{"reqId":"V9vXz2N4YUos7gbjeAO6zgAAAAY","remoteAddr":"192.168.0.124","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 423 \\\"DSR.xlsx\\\" is locked\",\"Exception\":\"OCA\\\\DAV\\\\Connector\\\\Sabre\\\\Exception\\\\FileLocked\",\"Code\":0,\"Trace\":\"#0 [internal function]: OCA\\\\DAV\\\\Connector\\\\Sabre\\\\LockPlugin->getLock(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/html\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(56): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/html\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/html\\\/o...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/dav\\\/lib\\\/Connector\\\/Sabre\\\/LockPlugin.php\",\"Line\":67,\"User\":\"sales8@sagar.com.np\"}","level":4,"time":"2016-09-16T11:30:23+00:00","method":"PUT","url":"\/remote.php\/webdav\/DSR.xlsx","user":"sales8@sagar.com.np"}
{"reqId":"V9vXzn5CiVMMJtGH97JhxAAAAAk","remoteAddr":"192.168.0.213","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:23+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sanjeev@sagar.com.np"}
{"reqId":"V9vXzn5CiVMMJtGH97JhxAAAAAk","remoteAddr":"192.168.0.213","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:23+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sanjeev@sagar.com.np"}
{"reqId":"V9vXzn5CiVMMJtGH97JhxAAAAAk","remoteAddr":"192.168.0.213","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:23+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sanjeev@sagar.com.np"}
{"reqId":"V9vXz0xH2bTYOUaRF56y9AAAAA8","remoteAddr":"192.168.0.124","app":"no app in context","message":"OCA\\Files_Antivirus\\AvirWrapper OCA\\Files_Antivirus\\AvirWrapper::fopen The antivirus executable could not be found at \/usr\/bin\/clamscan","level":2,"time":"2016-09-16T11:30:23+00:00","method":"PUT","url":"\/remote.php\/webdav\/Raju\/Corporate%20Visit%20Lists\/Corporate%20Client%20List.xlsx","user":"sales8@sagar.com.np"}
{"reqId":"V9vXz0xH2bTYOUaRF56y9AAAAA8","remoteAddr":"192.168.0.124","app":"no app in context","message":"OCA\\Files_Antivirus\\AvirWrapper OCA\\Files_Antivirus\\AvirWrapper::fopen The antivirus executable could not be found at \/usr\/bin\/clamscan","level":2,"time":"2016-09-16T11:30:23+00:00","method":"PUT","url":"\/remote.php\/webdav\/Raju\/Corporate%20Visit%20Lists\/Corporate%20Client%20List.xlsx","user":"sales8@sagar.com.np"}
{"reqId":"V9vX0XfYBTg11by9AoA4hQAAAAc","remoteAddr":"192.168.0.126","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:26+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sardak@sagar.com.np"}
{"reqId":"V9vX0XfYBTg11by9AoA4hQAAAAc","remoteAddr":"192.168.0.126","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:26+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sardak@sagar.com.np"}
{"reqId":"V9vX0XfYBTg11by9AoA4hQAAAAc","remoteAddr":"192.168.0.126","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:27+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sardak@sagar.com.np"}
{"reqId":"V9vX12N4YUos7gbjeAO60AAAAAY","remoteAddr":"192.168.0.149","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:33+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"jain@baleyo.com.np"}
{"reqId":"V9vX2TruiOCTO5weizGmOAAAAAA","remoteAddr":"192.168.0.71","app":"core","message":"File with id \"55578\" has not been found.","level":3,"time":"2016-09-16T11:30:34+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"buddhalaxmi@sagar.com.np"}
{"reqId":"V9vX2TruiOCTO5weizGmOAAAAAA","remoteAddr":"192.168.0.71","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:34+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"buddhalaxmi@sagar.com.np"}
{"reqId":"V9vX2klR9lsHPwQJA6MZTQAAABc","remoteAddr":"192.168.0.120","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:35+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"pujan@sagar.com.np"}
{"reqId":"V9vX2klR9lsHPwQJA6MZTQAAABc","remoteAddr":"192.168.0.120","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:35+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"pujan@sagar.com.np"}
{"reqId":"V9vX2ymTtdXCRktSWO58nwAAAAo","remoteAddr":"192.168.0.188","app":"core","message":"File with id \"42194\" has not been found.","level":3,"time":"2016-09-16T11:30:35+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"operation1@sagar.com.np"}
{"reqId":"V9vX2klR9lsHPwQJA6MZTQAAABc","remoteAddr":"192.168.0.120","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:35+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"pujan@sagar.com.np"}
{"reqId":"V9vX2n3rHZ7fwvhrJbA4BgAAAAQ","remoteAddr":"192.168.0.118","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:36+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sales7@sagar.com.np"}
{"reqId":"V9vX2n3rHZ7fwvhrJbA4BgAAAAQ","remoteAddr":"192.168.0.118","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:36+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sales7@sagar.com.np"}
{"reqId":"V9vX2n3rHZ7fwvhrJbA4BgAAAAQ","remoteAddr":"192.168.0.118","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:36+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sales7@sagar.com.np"}
{"reqId":"V9vX34UNGCUC6s7wLMXlZQAAAAU","remoteAddr":"192.168.0.117","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:40+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"homraj@sagar.com.np"}
{"reqId":"V9vX34UNGCUC6s7wLMXlZQAAAAU","remoteAddr":"192.168.0.117","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:40+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"homraj@sagar.com.np"}
{"reqId":"V9vX34UNGCUC6s7wLMXlZQAAAAU","remoteAddr":"192.168.0.117","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:40+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"homraj@sagar.com.np"}
{"reqId":"V9vX4Nf0e-E@r6a4Q0O9AQAAABQ","remoteAddr":"192.168.0.222","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:41+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"bijaykc@sagar.com.np"}
{"reqId":"V9vX4Nf0e-E@r6a4Q0O9AQAAABQ","remoteAddr":"192.168.0.222","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:41+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"bijaykc@sagar.com.np"}
{"reqId":"V9vX4NpWa6n59SMPgZMSwQAAAAs","remoteAddr":"192.168.0.169","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:41+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"basudev@sagar.com.np"}
{"reqId":"V9vX4VBF8bmKSa87gNwGLAAAABg","remoteAddr":"192.168.0.136","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:41+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"satya@sagar.com.np"}
{"reqId":"V9vX4Nf0e-E@r6a4Q0O9AQAAABQ","remoteAddr":"192.168.0.222","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:42+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"bijaykc@sagar.com.np"}
{"reqId":"V9vX5AspbZK-AfMDrKTzywAAABI","remoteAddr":"192.168.0.126","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:46+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sardak@sagar.com.np"}
{"reqId":"V9vX5AspbZK-AfMDrKTzywAAABI","remoteAddr":"192.168.0.126","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:46+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sardak@sagar.com.np"}
{"reqId":"V9vX5AspbZK-AfMDrKTzywAAABI","remoteAddr":"192.168.0.126","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:46+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sardak@sagar.com.np"}
{"reqId":"V9vX5cn4LNcaalILM5uCwgAAAAE","remoteAddr":"192.168.0.213","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:46+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sanjeev@sagar.com.np"}
{"reqId":"V9vX5cn4LNcaalILM5uCwgAAAAE","remoteAddr":"192.168.0.213","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:46+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sanjeev@sagar.com.np"}
{"reqId":"V9vX5cn4LNcaalILM5uCwgAAAAE","remoteAddr":"192.168.0.213","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:47+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"sanjeev@sagar.com.np"}
{"reqId":"V9vX7F5oqHdfSJDSSdn2CAAAABE","remoteAddr":"192.168.0.149","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:54+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"jain@baleyo.com.np"}
{"reqId":"V9vX7lBF8bmKSa87gNwGLQAAABg","remoteAddr":"192.168.0.136","app":"core","message":"File with id \"160875\" has not been found.","level":3,"time":"2016-09-16T11:30:55+00:00","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","user":"satya@sagar.com.np"}
{"reqId":"V9vX7tibaMtuNGHnf-GoWQAAAAM","remoteAddr":"192.168.0.71","app":"core","message":"File with id \"55578\" has not been found.","level":3,"time":"2016-09-16T11:30:56+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"buddhalaxmi@sagar.com.np"}
{"reqId":"V9vX7tibaMtuNGHnf-GoWQAAAAM","remoteAddr":"192.168.0.71","app":"core","message":"File with id \"61844\" has not been found.","level":3,"time":"2016-09-16T11:30:56+00:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"buddhalaxmi@sagar.com.np"}

.......
1) Tried reinstalling in client 2.2.3
2) Tried previous version in client 2.2.2
3) Turned off firewall and antivirus

After reinstalling the client software, connects to server but give "connection closed" msg while login.
Problem exist only two users yet with windows 10. Was fine a day before.

**Comment by RealRancor** [comment]
> Exception: {\"Message\":\"HTTP\\\/1.1 423 \\\"DSR.xlsx\\\" is locked

Related / duplicate of https://github.com/owncloud/core/issues/20380 ?

**Comment by PVince81** [comment]
If the connection is closed, it is more likely that the server ran into the shared storage hell recursive loop and ran out of memory and lost the connection. I see lots of messages related to sharing, so this is very likely. (ref: https://github.com/owncloud/core/issues/25557)

Please try again with 9.1.1 RC2 or wait for the final release next week.

**Comment by prabinpps** [comment]
Problem exist only with two users yet and for other users no problem. I have checked the memory, it is fine.

**Comment by RealRancor** [comment]
> Please try again with 9.1.1 RC2 or wait for the final release next week.

**Comment by prabinpps** [comment]
When RC2 will be released? 
I am getting another issue. When I log in I can access my files and admin page properly but users page opens and just gets rolling and rolling...

**Comment by RealRancor** [comment]
@prabinpps RC2 was already released a few days / weeks ago. The final version of 9.1.1 is scheduled for today.

> I am getting another issue. When I log in I can access my files and admin page properly but users page opens and just gets rolling and rolling...

Please don't mix several different issues in one bugreport. For support questions you should also use https://central.owncloud.org/ instead.

**Comment by prabinpps** [comment]
**fixed**: Upgraded to the 9.1.1 version
